source/transactions/tests/README.rst

```diff
@@ -454,6 +454,9 @@ prematurely timing out any tests' transactions. To decrease the timeout, run::
 
   db.adminCommand( { setParameter: 1, transactionLifetimeLimitSeconds: 3 } )
 
+Note that mongo-orchestration automatically sets this timeout to 3 seconds so
+drivers using mongo-orchestration do not need to run these commands manually.
```

Comment:
Might be wise to specify exactly what version of MO sets this, as some drivers could be using older versions.

Reply:
Done.